trestle/core/commands/command_docs.py

```diff
@@ -53,5 +54,9 @@ def _validate_arguments(self, args):
         if root is None:
             logger.error(f'Given directory {args.trestle_root} is not in a valid trestle root directory')
             return CmdReturnCodes.TRESTLE_ROOT_ERROR.value
+        is_oscal_dir_valid = fs.check_oscal_directories(args.trestle_root)
+        if not is_oscal_dir_valid:
+            logger.error('OSCAL directories are not valid, please remove extra files specified above.')
+            return CmdReturnCodes.TRESTLE_ROOT_ERROR.value
```

Comment:
This is an example where it is a user error and not an error in trestle.  I think we want that to be a warning and errors are only for something wrong internal.  But I'm flexible.

Reply:
I guess I wanted to emphasise to the user that this is an issue that needs to be fixed. The command will not proceed without this thus it is an error. But happy to make to log this at the INFO level if that is the convention we would like to follow.

Reply:
Normally I would do it as warning and not info - if the user made a mistake.  I use info for normal feedback the user needs - such as something being validated.  In this case given the need for the user to do something they may not be aware of - I think leaving it as error makes sense.

Reply:
Changed it to warning